### PR TITLE
[11.0][MIG] Knowledge Migration

### PR DESCRIPTION
**Pull request by fayezqandeel**

### CODE REVIEW

**Comment by fayezqandeel** [comment]
@pedrobaeza @mreficent I think now its fine, could check please?

**Review by MiquelRForgeFlow — APPROVED** [review]

File: knowledge/README.rst [hunk 1]
```diff
@@ -0,0 +1,75 @@
...
+Usage
+=====
+
+To use this module, you need to:
+
+* Go to Knowledge / Configuration / Settings
+
+.. image:: https://odoo-community.org/website/image/ir.attachment/5784_f2813bd/datas
+   :alt: Try me on Runbot
+   :target: https://runbot.odoo-community.org/runbot/repo/118/10.0
```
Comment (max3903) [inline knowledge/README.rst:30]: `/118/11.0`

File: knowledge/README.rst [hunk 2]
```diff
@@ -0,0 +1,75 @@
...
+Bug Tracker
+===========
+
+Bugs are tracked on `GitHub Issues <https://github.com/OCA/
+knowledge/issues>`_.
+In case of trouble, please check there if your issue has already been reported.
+If you spotted it first, help us smashing it by providing a detailed and welcomed feedback `here <https://github.com/OCA/
+knowledge/issues/new?body=module:%20
+knowledge%0Aversion:%20
+10.0%0A%0A**Steps%20to%20reproduce**%0A-%20...%0A%0A**Current%20behavior**%0A%0A**Expected%20behavior**>`_.
```
Comment (max3903) [inline knowledge/README.rst:47]: `11.0`

File: knowledge/README.rst [hunk 3]
```diff
@@ -0,0 +1,75 @@
...
+=======
+
+Contributors
+------------
+
+* Odoo SA <info@odoo.com>
+* Savoir-faire Linux <support@savoirfairelinux.com>
+* Gervais Naoussi <gervaisnaoussi@gmail.com>
+* Leonardo Donelli <leonardo.donelli@monksoftware.it>
+* Maxime Chambreuil <mchambreuil@ursainfosystems.com>
```
Comment (max3903) [inline knowledge/README.rst:60]: @fayezqandeel Please add yourself to this list

File: knowledge/data/ir_module_category.xml [hunk 4]
```diff
@@ -0,0 +1,12 @@
+<?xml version="1.0" encoding="utf-8"?>
```
Comment (max3903) [inline knowledge/data/ir_module_category.xml:1]: No need for the encoding any more.

**Review by max3903 — CHANGES_REQUESTED** [review]

File: knowledge/data/ir_module_category.xml (hunk 4 above)
Comment (pedrobaeza) [reply knowledge/data/ir_module_category.xml:1]: Yes for the XML files. It's not needed on Python files in Python 3.

**Comment by pedrobaeza** [comment]
Is this including a configuration flag for installing missing modules like CMIS? I don't see that like a good idea.

**Comment by max3903** [comment]
@pedrobaeza Yes it does. I think it's a good way to promote other related modules.

**Comment by pedrobaeza** [comment]
Yes, but not know that the rest of the modules are not migrated (and maybe they will be never migrated) and thus they throw an error. This should be detected dynamically if there's a module in the module list with the proper name for showing it.

**Comment by max3903** [comment]
@fayezqandeel Are you going to fix this pull request? Please let us know as there is a lot of other modules depending on this one. Thanks.

**Comment by cubells** [comment]
Hi @fayezqandeel 

I want to finish this pr. Can I push to this branch?

**Comment by SalahAdDin** [comment]
@cubells we need too.

**Comment by cubells** [comment]
@SalahAdDin I'll create a new pr.

**Comment by SalahAdDin** [comment]
@cubells ok, i'm waiting.

**Comment by cubells** [comment]
@saadthaifa new pr is here: https://github.com/OCA/knowledge/pull/170 and supersede this.

**Pull request closed without merging**